scripts/veCompare/veCompareViewFunctions.js

```diff
@@ -10,8 +10,6 @@ async function main() {
     const twoOLASBalance = ethers.utils.parseEther("200");
     const threeOLASBalance = ethers.utils.parseEther("300");
     const fiveOLASBalance = ethers.utils.parseEther("500");
-    // Set the original or wrapped veOLAS
-    const checkOriginalVE = true;
```

Comment:
Since wrapped veOLAS is there, no need to set limits anymore, with wrapped veOLAS we can have all the same set of tests as with the original curve Voting Escrow